src/rollup/types.d.ts

```diff
@@ -518,6 +524,7 @@ export interface RenderedChunk extends PreRenderedChunk {
 }
 
 export interface OutputChunk extends RenderedChunk {
+	type: 'chunk';
```

Comment:
another option is to add:
```
isAsset: false;
```
in OutputChunk.

So type discrimination can be performed with isAsset directly.

Reply:
Indeed. you right.
But it does not seem semantically pleasing:)
Especially since https://github.com/rollup/rollup/blob/2443783d0b1214ab742373414ed03d4f0f75964f/src/utils/FileEmitter.ts#L62

Already has this pattern implemented.
So maybe this will lead to some eventual consistency in types.

Does it make sense?

Reply:
@manucorporat `isAsset` workarround will be the easiest fix. for the issue.

If you agree that some technical debt is there to make data and types consistent across the project,  I will be willing to contribute towards that